superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -261,22 +261,31 @@ export function handleDashboardDelete(
   );
 }
 
+export function shortenSQL(sql: string, maxLines: number) {
+  let lines: string[] = sql.split('\n');
+  if (lines.length >= maxLines) {
+    lines = lines.slice(0, maxLines);
+    lines.push('...');
```

Comment:
The `...` is probably not necessary, since it will be pushed down to hidden anyway if any of the SQL lines was wrapped:

![image](https://user-images.githubusercontent.com/335541/100053179-59552100-2dd4-11eb-8f32-3185ad53ff67.png)

Can probably just set a higher `maxLines` and make sure there's no unnecessary blank space at the bottom of the SQL code area.

Reply:
if the line-wrapping is causing the extra lines, I would be open to any of these options:
1) Not worrying about it (though a super-duper long line might get crazy)
2) Not letting it wrap (CSS e.g. `white-space: nowrap`)
3) Cropping off the query with a max-height
4) ... dare I say scrolling?

Personally I'd vote for #2 or #2

Reply:
Personally I'd say let it wrap and just get rid of "...".